jenkins/Jenkinsfile

```diff
@@ -118,6 +118,9 @@ pipeline {
             sh '''
               juliaup update
             '''
+            sh '''
+              julia -e 'using Pkg; Pkg.activate("./NDTensors/test/"); Pkg.add("Metal")';
```

Comment:
This fix seems strange to me, since the tests already execute `Pkg.add("Metal")` when `Pkg.test("NDTensors"; test_args=["metal"])` is called.